Makefile

```diff
@@ -10,7 +10,7 @@ SELF:=$(firstword $(MAKEFILE_LIST))
 
 PKG=hgvs
 PKGD=$(subst .,/,${PKG})
-TEST_DIRS:=.,doc,hgvs,tests
```

Comment:
I'm not sure what effect the `.` was intended to have, but it caused pytest to have issues, and didn't seem to affect what tests were actually run during `make test` between master and this branch.